src/tests/test_utils.ts

```diff
@@ -9,116 +9,99 @@ import {
 } from "../types.ts";
 
 export const FAKE_ID = "ABC123";
-export const generatePayload = (
-  id: string,
+export const generateFunctionExecutedPayload = (
+  callback_id: string,
+  enterprise_id?: string,
 ): InvocationPayload<FunctionInvocationBody> => {
   return {
     body: {
       event: {
         type: "function_executed",
-        function: { callback_id: id },
+        function: { callback_id },
         function_execution_id: FAKE_ID,
         bot_access_token: FAKE_ID,
         inputs: {},
       },
-      enterprise_id: FAKE_ID,
+      // Setting to undefined in an effort to model how this property shows up in production: either exists, or it is not defined at all
+      enterprise_id: enterprise_id ? enterprise_id : undefined,
     },
     context: { bot_access_token: FAKE_ID, team_id: FAKE_ID, variables: {} },
   };
 };
 
-export const generateBaseInvocationBody = (
+export const generateBaseEventInvocationBody = (
   type: string,
-  id?: string,
+  callback_id?: string,
+  enterprise_id?: string,
 ): InvocationPayload<BaseEventInvocationBody> => {
-  return {
+  const payload = {
     body: {
       type,
       function_data: {
         execution_id: FAKE_ID,
-        function: { callback_id: id || FAKE_ID },
+        function: { callback_id: callback_id || FAKE_ID },
         inputs: {},
       },
       bot_access_token: FAKE_ID,
     },
     context: { team_id: FAKE_ID, bot_access_token: FAKE_ID, variables: {} },
-  };
+  } as InvocationPayload<BaseEventInvocationBody>;
+  if (enterprise_id) {
+    payload.body.enterprise = { id: enterprise_id };
+  }
+  return payload;
 };
 
 export const generateBlockActionsPayload = (
-  id?: string,
+  callback_id?: string,
+  enterprise_id?: string,
 ): InvocationPayload<BlockActionInvocationBody> => {
-  return {
-    body: {
-      type: "block_actions",
-      actions: [],
-      function_data: {
-        execution_id: FAKE_ID,
-        function: { callback_id: id || FAKE_ID },
-        inputs: {},
-      },
-      enterprise: { id: FAKE_ID },
-      bot_access_token: FAKE_ID,
-    },
-    context: { bot_access_token: FAKE_ID, team_id: FAKE_ID, variables: {} },
-  };
+  const payload = generateBaseEventInvocationBody(
```

Comment:
A little refactor in the test utils to re-use the `generateBaseEvent` helper for all of the block actions / view event payload tests.